.github/workflows/test_integration_adapter.yml

```diff
@@ -64,6 +65,11 @@ jobs:
             pip install 'dbt-${{ matrix.profile }}==${{ matrix.adapter_version }}'
           fi
 
+          if [[ '${{ matrix.profile }}' == "redshift" ]]
+          then
+            pip install dbt-postgres
+          fi
+
```

Comment:
I think we don't have to install it, it's a dependency: https://github.com/dbt-labs/dbt-redshift/blob/main/setup.py#L70

Reply:
Good catch!